DESCRIPTION

```diff
@@ -35,5 +35,6 @@ SystemRequirements: Python (>= 2.7), QGIS (>= 2.12)
 Suggests: 
           knitr,
           rmarkdown,
-          testthat
+          testthat,
+          sf
```

Comment:
That's right. However, you need to import the function in the doc to be able to use it and you can only import functions from packages which are listed under 'imports:' in the DESCRIPTION